features/grid.yml.dist

```diff
@@ -157,6 +179,21 @@ compat_features:
   #   safari_ios: "11"
   - css.properties.place-content.grid_context
 
+  # baseline: high
+  # baseline_low_date: ≤2020-03-24
+  # baseline_high_date: ≤2022-09-24
+  # support:
+  #   chrome: "57"
+  #   chrome_android: "57"
+  #   edge: "79"
+  #   firefox: "52"
+  #   firefox_android: "52"
+  #   safari: ≤13.1
+  #   safari_ios: ≤13.4
+  - css.properties.grid-auto-flow.column
+  - css.properties.grid-auto-flow.dense
+  - css.properties.grid-auto-flow.row
```

Comment:
These version ranges make no sense. I sent https://github.com/mdn/browser-compat-data/pull/24539 to fix this so it matches the other Safari 10.1 items.

Reply:
@ddbeck  I've updated the dist with the BCD changes here, and it looks much better.